Source/DafnyCore/AST/Modules/ModuleDefinition.cs

```diff
@@ -1078,6 +1078,8 @@ bool InheritsFromObject(TraitDecl traitDecl) {
   });
 
   public SymbolKind? Kind => SymbolKind.Namespace;
+  [CanBeNull] public LiteralModuleDecl EnclosingLiteralModuleDecl { get; set; }
```

Comment:
```suggestion
  public LiteralModuleDecl? EnclosingLiteralModuleDecl { get; set; }
```
Doesn't that do the same, but without the extra import?